From: Jinjiang Tu <tujinjiang@huawei.com>
Date: Mon, 24 Aug 2026 14:10:09 +0800
Subject: [PATCH] docs: ksm: fix typos in sysfs knob names

Patch series "docs/ksm: fix advisor documentation and comment", v3.

This series fixes two problems left in the KSM advisor documentation
and code comment:
 - Patch 1 fixes two typos in sysfs knob names ("adivsor_max_cpu" and
   "adivsor_max_pages_to_scan") in ksm.rst that don't match the actual
  knob names.
 - Patch 2 fixes the description of advisor_min_pages_to_scan: it is
   described as the lower limit of pages_to_scan, but is actually only
   used as it's initial value, and the runtime pages_to_scan can drop
   below advisor_min_pages_to_scan.


This patch (of 2):

The sysfs knob names in mm/ksm.c are "advisor_max_cpu" and
"advisor_max_pages_to_scan", but the ksm.rst documentation spelled both as
"adivsor_*", fix the two typos.

Link: https://lore.kernel.org/20260824061010.3343959-1-tujinjiang@huawei.com
Link: https://lore.kernel.org/20260824061010.3343959-2-tujinjiang@huawei.com
Signed-off-by: Jinjiang Tu <tujinjiang@huawei.com>
Acked-by: David Hildenbrand (Arm) <david@kernel.org>
Reviewed-by: Lorenzo Stoakes (ARM) <ljs@kernel.org>
Acked-by: Randy Dunlap <rdunlap@infradead.org>
Cc: Chengming Zhou <chengming.zhou@linux.dev>
Cc: Jonathan Corbet <corbet@lwn.net>
Cc: Kefeng Wang <wangkefeng.wang@huawei.com>
Cc: Liam R. Howlett <liam@infradead.org>
Cc: Michal Hocko <mhocko@suse.com>
Cc: Mike Rapoport <rppt@kernel.org>
Cc: Nanyong Sun <sunnanyong@huawei.com>
Cc: Stefan Roesch <shr@devkernel.io>
Cc: Suren Baghdasaryan <surenb@google.com>
Cc: Vlastimil Babka <vbabka@kernel.org>
Cc: xu xin <xu.xin16@zte.com.cn>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---
 Documentation/admin-guide/mm/ksm.rst | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Documentation/admin-guide/mm/ksm.rst b/Documentation/admin-guide/mm/ksm.rst
index ad8e7a41f3b5d..c9f533b10f6f9 100644
--- a/Documentation/admin-guide/mm/ksm.rst
+++ b/Documentation/admin-guide/mm/ksm.rst
@@ -174,7 +174,7 @@ advisor_mode
         The section about ``advisor`` explains in detail how the scan time
         advisor works.
 
-adivsor_max_cpu
+advisor_max_cpu
         specifies the upper limit of the cpu percent usage of the ksmd
         background thread. The default is 70.
 
@@ -186,7 +186,7 @@ advisor_min_pages_to_scan
         specifies the lower limit of the ``pages_to_scan`` parameter of the
         scan time advisor. The default is 500.
 
-adivsor_max_pages_to_scan
+advisor_max_pages_to_scan
         specifies the upper limit of the ``pages_to_scan`` parameter of the
         scan time advisor. The default is 30000.
 
-- 
2.47.3

